lib/db.js

```diff
@@ -215,19 +213,24 @@ export default {
     }
   },
 
-  put: async function(obj, table) {
+  put: async function(obj, table, createNew) {
+    let conditionExpression = "attribute_exists(id)";
+    if (createNew) {
+      conditionExpression = "attribute_not_exists(id)";
+    }
     const params = {
       Item: obj,
       TableName: table + (process.env.ENVIRONMENT || ""),
-      ConditionExpression: "attribute_not_exists(id)"
+      ConditionExpression: conditionExpression
     };
 
     try {
-      const res = await docClient.put(params).promise();
-      return res
+      const command = new PutCommand(params);
+      const res = await docClient.send(command);
+      return res;
     } catch (err) {
       const errorResponse = this.dynamoErrorResponse(err);
-      throw errorResponse
+      throw errorResponse;
```

Comment:
I'm curious about the searchability and formatting of this exception on the CloudWatch logs compared to error logging? I've found other projects will also have a console.error or similar before throwing an exception, so I'm not sure if we're benefiting from removing all our error logging - lmk

Reply:
oops didnt mean to delete the ddb error log